src/test/java/seedu/address/model/order/DateTest.java

```diff
@@ -53,4 +53,15 @@ public void hashcode() {
         // different date -> returns different hashcode
         assertNotEquals(date.hashCode(), new Date("2020-05-01").hashCode());
     }
+
+    @Test
+    public void compareTo() {
+        Date earlierDate = new Date("2020-01-01");
+        Date laterDate = new Date("2020-02-01");
+
+        assertTrue(earlierDate.compareTo(laterDate) < 0, "Expected earlierDate to be less than laterDate");
+        assertTrue(laterDate.compareTo(earlierDate) > 0, "Expected laterDate to be greater than earlierDate");
+        assertEquals(0, earlierDate.compareTo(new Date("2020-01-01")),
+                "Expected dates with the same value to be equal");
```

Comment:
Currently for all the test cases we have, there is no message for the test. Would it be better to remove them for  standarisation?